osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/FirmwareManagementServiceTest.java

```diff
@@ -552,4 +582,222 @@ void checkSsldPendingFirmwareUpdateReturnsTrueAndDeletesPendingUpdateWithMatchin
 
         verify(this.ssldPendingFirmwareUpdateRepository).delete(matchingPendingFirmwareUpdate);
     }
+    
+    
+    /*
+     * Returns basic CorrelationIds for simple tests
+     */
+    private CorrelationIds getCorrelationIds() {
+    	final String organisationIdentification = "test-org";
+        final String deviceIdentification = "device-identification";
+        final String correlationUid = "correlation-uid";
```

Comment:
Since we are going to check for these values in the assert part of the tests, refactor them into constants.